src/diff/children.js

```diff
@@ -82,6 +82,11 @@ export function diffChildren(parentDom, newParentVNode, oldParentVNode, context,
 				}
 			}
 
+			if (lastRendered && lastRendered._component) {
+				lastRendered._component._siblingVNode = childVNode;
```

Comment:
Could we add the `_sibling` pointer to the VNode? Would remove a check and a property access.
```js
// diffChildren
if (lastRendered) {
	lastRendered._sibling = childVNode;
}
lastRendered = childVNode;
```

```js
// component.js
let sibling = vnode._sibling;
```